Packs/TIMCampaignTracking/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "TIM Campaign Tracking",
+    "description": "Upload threat intelligence briefs to track IOCs related to specified threat actor campaigns.",
```

Comment:
"This pack allows you to upload threat intelligence briefs to track IOCs related to specified threat actor campaigns. Using this pack you will be able to check what the adversary is trying to accomplish, why are they trying to accomplish this, and so on"